css/site.styl

```diff
@@ -108,11 +108,11 @@ $margin-basic = ($gutter * 2)
 $margin-mid = ($gutter * 3.5)
 $margin-large = ($gutter * 5)
 
-$font-size-basic = 15px
+$font-size-basic = 13px
 $font-size-larger = 1.3rem
-$font-size-h3 = golden-ratio(1rem , 1)
-$font-size-h2 = golden-ratio(1rem , 1.5)
-$font-size-h1 = golden-ratio(1rem , 2)
+$font-size-h3 = golden-ratio(1rem , .3)
```

Comment:
but h3 is already used for post titles in the grid. this is going to make those titles the wrong size no? h3 isnt even used in the header so what is the change here for?